src/@newrelic/gatsby-theme-newrelic/components/GlobalFooter.js

```diff
@@ -91,27 +83,26 @@ const GlobalFooter = ({
           />
         </Link>
         <div>
-          {repository && (
-            <CreateIssueButton
-              pageTitle={pageTitle}
-              variant={Button.VARIANT.OUTLINE}
-              size={Button.SIZE.SMALL}
-              labels={issueLabels}
-              instrumentation={{ component: 'GlobalFooter' }}
+          <Button
+            as={Link}
```

Comment:
Looks like this button is trying to do a relative link for some reason. I think this might need to be an `ExternalLink`
```suggestion
            as={ExternalLink}
```